geth-with-protocol/Dockerfile

```diff
@@ -36,9 +35,9 @@ RUN ./build-protocol.sh
 
 # Now create clean image
 FROM alpine:latest
-ENV GETH_MINING_ACCOUNT 87da6a8c6e9eff15d703fc2773e32f6af8dbe301
+ENV GETH_MINING_ACCOUNT 0161e041aad467a890839d5b08b138c1e6373072
 ENV gethRoot /root/.ethereum
-LABEL org.livepeer.controller="0xA1fe753Fe65002C22dDc7eab29A308f73C7B6982"
+LABEL org.livepeer.controller="0x77A0865438f2EfD65667362D4a8937537CA7a5EF"
```

Comment:
Why did this change?

Reply:
We are using a different account (whatever $GETH_MINING_ACCOUNT is) to deploy the contracts so the contract address changes as it deterministic but one of the params is the sender of the tx. 

The previous value of the ENV variable didn't correctly represent what it semantically says. 

> In Dockerfile changed GETH_MINING_ACCOUNT env variable to be the correct sealer (0x0161e0...) as defined in genesis.json extradata field, 0x87da6... is a funded genesis account but not the sealer.

Reply:
Got it. Might be a good idea to change the default Controller address in the example config files in the test-harness as well then.